### ISSUE TITLE
Threshold option broken?

### ISSUE THREAD

**Issue by stefanjudis** [addyosmani/psi#48]
Hey hey, 

I just wanted to play around with the `--threshold` option, but unfortunately it is not working.

I'm running

```
> psi www.smashingmagazine.com --threshold=99                                               [9:50:44]

--------------------------------------------------------

URL:       smashingmagazine.com
Strategy:  mobile
Score:     98

CSS size                                   | 559.91 kB
HTML size                                  | 65.81 kB
Image size                                 | 385.25 kB
JavaScript size                            | 161.88 kB
CSS resources                              | 3
Hosts                                      | 8
JS resources                               | 8
Resources                                  | 55
Static resources                           | 32
Total size                                 | 8.07 kB

Leverage browser caching                   | 0.75
Optimize images                            | 0.28

--------------------------------------------------------
```

I already a had look into the code, and there is no special treatment for `threshold` inside of psi and also the docs about the API doesn't say anything about threshold handling?

https://developers.google.com/speed/docs/insights/v1/getting_started#examples

Do I miss something?

**Comment by sindresorhus** [comment]
Oops, I seem to have forgotten to handle that case in the recent rewrite. Fixed in 1.0.5.

```
npm i -g psi
```

**Comment by stefanjudis** [comment]
Ha - thanks. I just wanted to propose the same PR. :)

But anyway - thanks for super quick response. :bowtie:
